src/sentry/event_manager.py

```diff
@@ -525,12 +525,10 @@ def save_error_events(
         secondary_hashes = None
         migrate_off_hierarchical = False
 
-        if _should_run_secondary_grouping(project):
+        if should_run_secondary_grouping(project):
             with metrics.timer("event_manager.secondary_grouping", tags=metric_tags):
                 secondary_grouping_config = SecondaryGroupingConfigLoader().get_config_dict(project)
-                secondary_hashes = _calculate_secondary_hash(
-                    project, job, secondary_grouping_config
-                )
+                secondary_hashes = calculate_secondary_hash(project, job, secondary_grouping_config)
```

Comment:
Q: wouldn't it make sense to roll `should_run_secondary_grouping` and `calculate_secondary_hash`, have the latter short circuit to return `None` when int shouldn't calculate the hash? 

I don't think these two functions are ever used separately, apart from tests.